src/enhancers/layout.ts

```diff
@@ -51,9 +51,9 @@ export const propEnhancers: PropEnhancers = {
   boxSizing: (value: PropEnhancerValueType) => getCss(boxSizing, value),
   clear: (value: PropEnhancerValueType) => getCss(clear, value),
   clearfix: () => ({
-    className: '📦clearfix',
+    className: 'ui_box_clearfix',
```

Comment:
in favor of brevity here I would suggest `box-*`, it's double the chars that the emoji is (`📦`= 2 vs `box-` = 4)

Reply:
I could also do a ❒ as someone suggested in the issues. It won't take on native styling so it'd stick out much less in the DOM.

Reply:
You could also use `ub-*` (`ui-box-*`), as `box-*` sounds common enough to run into conflicts.

Reply:
added `ub-`